src/tests/pmempools/pmempool_create/pmempool_create.cc

```diff
@@ -38,7 +38,7 @@ int PmempoolCreate::CreatePool(const PoolArgs &pool_args,
       struct_utils::POOL_TYPES[static_cast<int>(pool_args.pool_type)];
   create_args += struct_utils::CombineArguments(pool_args.args) + " " + path;
 
-  output_ = shell_.ExecuteCommand("pmempool create " + create_args);
+  output_ = shell_.ExecuteCommand(bins::pmempool + " create " + create_args);
```

Comment:
I know we discussed this before, but I after seeing this patch I realized that you could add dirname(BINS_PMEMPOOL) to PATH (using set_tests_properties) and get rid of "bins" namespace.